midend/actionSynthesis.cpp

```diff
@@ -60,7 +60,7 @@ const IR::Node* DoMoveActionsToTables::postorder(IR::MethodCallStatement* statem
         ++arg;
     }
     BUG_CHECK(arg == mc->arguments->end(), "%1%: mismatched arguments", mc);
-    auto amce = new IR::MethodCallExpression(mc->srcInfo, mc->method, mc->typeArguments, otherArgs);
+    auto amce = new IR::MethodCallExpression(mc->srcInfo, actionPath, mc->typeArguments, otherArgs);
```

Comment:
why change this?

Reply:
The name of an action in action list is different from the same action in default action, e.g.
```
action = { action_1; }
default_action = action_1/action;
```

The name of the action has original name, but the action in action list has not.

I reverted the change in this PR, since I do not need the actionSynthesis pass in dpdk backend anyway.